pkg/standalone/testdata/runfileconfig/test_run_config_empty_app_dir.yaml

```diff
@@ -1,24 +1,24 @@
 version: 1
 common:
-  resources_path: ./app/resources
-  app_protocol: HTTP
-  app_health_probe_timeout: 10
+  resourcesPath: ./app/resources
+  appProtocol: HTTP
+  appHealthProbeTimeout: 10
   env:
     - name: DEBUG
```

Comment:
@pravinpushkar please fix this unit test and remove these fields if not necessary. 
please fix in #1181

Reply:
addressed